plugins/obs-ffmpeg/obs-ffmpeg-mpegts.c

```diff
@@ -598,9 +607,9 @@ bool ffmpeg_mpegts_data_init(struct ffmpeg_output *stream, struct ffmpeg_data *d
 
 /* ------------------------------------------------------------------------- */
 
-static inline bool stopping(struct ffmpeg_output *output)
+static inline bool stopping(struct ffmpeg_output *stream)
```

Comment:
I'm as guilty as anyone for doing this; but extensive renaming combined with many logic changes makes it harder to review the commit; would be great to separate them into two commits in future.

Reply:
haha, that's fair. Followed your advice and split the renaming to its own commit; the renaming is because the logging function expects the struct to be named 'stream' but I was careless because i wrote the output as a mongrel between obs-ffmpeg-output and rtmp-stream.